src/styles/_shell.scss

```diff
@@ -42,14 +42,14 @@
       font-family: $font-family;
     }
   }
-  :global(#overlay-portal) {
+  :global(.overlay-portal) {
```

Comment:
check if we can remove the global - as now we have the portal only under a player

Reply:
cannot remove it